### ISSUE TITLE
'module' object has no attribute 'control_flow_ops' 

### ISSUE THREAD

**Issue by Pranav-Goel** [shyamupa/snli-entailment#5]
I get this error when trying to run amodel.py - I use keras with tensorflow backend (keras v1.0.6) and I have tensorflow v0.12.1 (cannot get 0.11 versions now as they are no longer available)

However, I am able to run tf_model.py -> with regards to this code, how can I get the final predicted labels (neutral/entailment/contradiction) on test data (and not just the test accuracy). I am trying to do simple changes to the code to get the predictions as well (through tf,argmax(self.pred,1) and then using eval function. But I am pretty new to tensorflow and am struggling with this. I would be very grateful if you could help me with this as well. What I want is the final prediction array (0/1/2) for all the test instances.

Thank you.
